src/applications/caregivers/definitions/sharedUI.js

```diff
@@ -0,0 +1,112 @@
+import { merge } from 'lodash';
+import get from 'platform/utilities/data/get';
+import VaTextInputField from 'platform/forms-system/src/js/web-component-fields/VaTextInputField';
+import {
+  radioUI,
+  selectUI,
+  dateOfBirthUI,
+  addressUI as platformAddressUI,
+  fullNameUI as fullNameWithSuffixUI,
+  phoneUI as platformPhoneUI,
+  emailUI as platformEmailUI,
+  ssnUI as platformSsnUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { genderLabels } from 'platform/static-data/labels';
+import { validateSsnIsUnique } from '../utils/validation';
+import { replaceStrValues } from '../utils/helpers';
+import AddressWithAutofill from '../components/FormFields/AddressWithAutofill';
+import CustomReviewField from '../components/FormReview/CustomReviewField';
+import content from '../locales/en/content.json';
+
+export const addressUI = props => {
+  const {
+    label,
+    hint = null,
+    requireCounty = false,
+    countyDescription = null,
+  } = props;
+  return merge({}, platformAddressUI({ omit: ['isMilitary', 'street3'] }), {
```

Comment:
We need to override the default web component pattern to force the country field to default to the United States. We only allow U.S. addresses.